frame/tips/src/lib.rs

```diff
@@ -379,6 +383,31 @@ decl_module! {
 			Tips::<T>::insert(&hash, tip);
 		}
 
+		/// Remove and slash an already-open tip.
+		///
+		/// May only be called from `T::RejectOrigin`.
+		///
+		/// As a result, API will slash the finder and the deposits are lost.
+		///
+		/// Emits `TipSlashed` if successful.
+		#[weight = 10_000]
```

Comment:
You need to write a new benchmark for this, and use the `T::WeightInfo` trait like all the other functions.